Adding Details to the Installation Guide

RubberNoodles opened the pull request:
Specifically a bit more detail on how to edit mup.js and which parts of the file would be important to change for deployment on a different public server.

edemaine commented:
Thanks for the suggestions!

Do you have documentation for the M1 claim? Did you have to follow any of the steps [listed here](https://forums.meteor.com/t/solved-install-meteor-new-processor-mac/54559)?

There are lots of small typos and unnecessary changes here, which you can see if you look at the diff.  I'm also pretty sure that the indentation and lists are off for Markdown; did you test?  I can make these edits if you can't.

RubberNoodles commented:
Yup! I was forwarded from Michael at OOEP. Here was the issue that I faced while trying to install Meteor using an Ubuntu virtual machine via Parallels Desktop on my ARM device (Mac with M1 chip). 

<img width="712" alt="Screen Shot 2021-01-30 at 6 28 00 PM" src="https://user-images.githubusercontent.com/69167715/106372040-e498e880-6328-11eb-8db9-1f2975430fd3.png">

I haven't tried it myself, but yes! The link you gave was what I was referencing when I added "Rosetta for machines on an M1 Chip"!

Also, oops! I deleted the unnecessary changes. I'm totally new to markdown, so I wasn't quite sure what you mean by test. When I look at it from the preview on GitHub it seems to look alright, but I am probably confused.
